css/src/components/form/input.scss

```diff
@@ -0,0 +1,84 @@
+$input-color: $text !default;
+$input-background-color: $body-background !default;
+$input-border-color: $text !default;
+$input-placeholder-color: $text-subtle !default;
+
+$input-hover-color: $text !default;
+$input-hover-border-color: $primary !default;
+$input-focus-border-color: $primary !default;
+
+$input-disabled-color: $text-subtle !default;
+$input-disabled-background-color: $body-background-medium !default;
+$input-disabled-border-color: $table-border-dark !default;
+
+$input-error-border-color: $danger !default;
+$input-warning-border-color: $warning-active !default;
+$input-success-border-color: $score-high !default;
+
+$input-focus-box-shadow-size: 0 0 0 0.125rem !default;
+
+.input {
+	@include control;
+
+	width: 100%;
+	max-width: 100%;
+	border-color: $input-border-color;
+	background-color: $input-background-color;
+	color: $input-color !important;
+	margin-block-start: 0.5rem;
```

Comment:
don't think we require/should have margin here

Reply:
I added it here thinking about how the input comes with a label or the case when a bunch of form components comes one after another. But I can definitely see the cases when it might be not needed, so will remove it. However, I think the label component needs to have it by default instead.

Reply:
Removed